### DISCUSSION TITLE
Adding template YML format for PR

### DISCUSSION THREAD

**Discussion by thomasbnt** [community/community#53360] [Pull Requests]
### Select Topic Area

General

### Body

Like a [issue template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms), have a possibility to have templates in YML format.

**Comment by queenofcorgis** [comment]
Hi there - thanks for the feedback - I've added the proper label to this so our team can review. Have you used [Pull Requests templates in markdown](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) at all? If so I'd love to hear your experience and why you would prefer a YML format. Feel free to share any use cases as well.

**Reply by thomasbnt** [reply]
Hello ! 

I prefer using YML format because the text fields are quicker to fill in

**Comment by Savio-Sou** [comment]
Seemingly duplicates:
- https://github.com/orgs/community/discussions/4527
- https://github.com/orgs/community/discussions/27897
- https://github.com/orgs/community/discussions/42973
